app/assets/stylesheets/v3/layout/variables.scss

```diff
@@ -50,7 +50,7 @@ $blue-main-brand--lighter: #bdd3e5;
 $bg-blue-light: #ddeaf3;
 $bg-grey-lightest: #f4f4f4;
 $bg-grey-lighter: #e5e5e5;
-$bg-grey-light: #dbdbdb;
```

Comment:
I know you are still working on this, but I think this comes from the QA site variables: 

https://qa.ama.ab.ca/styleguide

Do they have a different value elsewhere?

Reply:
I got it from the frontify styleguide because earlier in this project Kayt had said to go off of that one.